tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cuda11_1_tensorrt7_2

```diff
@@ -106,14 +100,14 @@ RUN export SWIG_ROOT=swig-4.0.2 && \
 
 FROM build_base AS build_cpython
 COPY build_scripts/build-sqlite3.sh /build_scripts/
-RUN export SQLITE_AUTOCONF_ROOT=sqlite-autoconf-3360000 && \
-    export SQLITE_AUTOCONF_HASH=bd90c3eb96bee996206b83be7065c9ce19aef38c3f4fb53073ada0d0b69bbce3 && \
-    export SQLITE_AUTOCONF_DOWNLOAD_URL=https://www.sqlite.org/2021 && \
+RUN export SQLITE_AUTOCONF_ROOT=sqlite-autoconf-3370200 && \
```

Comment:
can we put these variable definitions into a common place so that we don't need to update different files when the version changes?

e.g., `RUN source export_sqlite_vars.sh && manylinux-entrypoint /build_scripts/build-sqlite3.sh`

export_sqlite_vars.sh:
```
export SQLITE_AUTOCONF_ROOT=sqlite-autoconf-3370200
export SQLITE_AUTOCONF_HASH=4089a8d9b467537b3f246f217b84cd76e00b1d1a971fe5aca1e30e230e46b2d8 && \
export SQLITE_AUTOCONF_DOWNLOAD_URL=https://www.sqlite.org/2022
```

we'd also need to update the cgmanifest file generation

Reply:
I just tried it. Not very good. Currently I just copy and paste the code from manylinux repo to ours. If there are too many Dockerfiles to modify, I can make a script.  I already put makers in the docker files.
```
#Build manylinux2014 docker image begin
...
#Build manylinux2014 docker image end
```
If I extract the export commands to a separated file, I think it will need more human work to maintain.

Reply:
Can docker file include another docker file?

Reply:
No. https://github.com/moby/moby/issues/735

Reply:
Could you elaborate on how it's more work to maintain?
I counted 11 instances of SQLITE_AUTOCONF_ROOT in the changes (different Dockerfiles). Changing it in one place seems less error-prone and easier to review.
Updating them with a script is a good idea too.